docs/semgrep-appsec-platform/gitlab-mr-comments.md

```diff
@@ -107,12 +107,11 @@ For **other CI providers**:
 For more configuration options, see [GitLab CI Sample](/semgrep-ci/sample-ci-configs#gitlab-cicd).
 </details>
 
-:::info MR comments with multiple GitLab groups
-If you're using Semgrep with multiple GitLab groups, ensure that you've completed one of the following steps to ensure that you see MR comments for repositories associated with all of your groups:
+#### MR comments with multiple GitLab groups
+If you're using Semgrep with multiple GitLab groups, ensure that you've completed the following steps to ensure that you see MR comments for repositories associated with all of your groups:
 
-1. Create a Semgrep account connection with each GitLab group by signing in to your Semgrep account, navigating to **Settings > Source code managers**, and adding an entry for each group. Note that you only need to add an entry for the parent group to onboard a parent group and all of its subgroups.
+1. **Required for users with GitLab self-managed plans and *optional* for users with GitLab Cloud plans:** Create a Semgrep account connection with each GitLab group by signing in to your Semgrep account, navigating to **Settings > Source code managers**, and adding an entry for each group. Note that you only need to add an entry for the parent group to onboard a parent group and all of its subgroups. 
```

Comment:
```suggestion
1. **Required for users with GitLab self-managed plans and *optional* for users with GitLab Cloud plans:** Create a Semgrep source code connection with each GitLab group by signing in to your Semgrep account, navigating to **Settings > Source code managers**, and adding an entry for each group. Note that you only need to add an entry for the parent group to onboard a parent group and all of its subgroups.
```

Reply:
I don't know if "source code connection" is ideal here, but "account connection" sounds strange to me.

Reply:
I wonder if we can use source code manager (SCM) connection? We use the term "connect/connection" in the Settings UI itself, and the tab is named **Source code managers**. See: https://semgrep.dev/orgs/-/settings/source-code